etc/home-env-funcs.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+sirepo_add_admin() {
+    declare u
+    # POSIT: only uids in user and they don't have any specials
+    for u in $(ls ~/src/radiasoft/sirepo/run/user); do
```

Comment:
I think these functions should be sirepo_dev_add_admin.

`~/src/radiasoft` and `~/mail` should be variables.

Reply:
9cf9e4cee